### PR TITLE
Adding WOSPM Checker

### PR DESCRIPTION
**Pull request by umutphp**
Even if it is in early stage, I think my project is good enough to list in Up For Graps lists

### CODE REVIEW

File: _data/projects/wospm-checker.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: WOSPM Checker
+desc: A checker for open source project owners to measure their open source project if welcoming or not
```
Comment (ritwik12) [inline _data/projects/wospm-checker.yml]: @umutphp Can we improve the Description? last few lines are not properly understood

File: _data/projects/wospm-checker.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+site: https://wospm.github.io/checker/
+tags:
+  - php
+  - open-source
+  - oss
+  - metrics
+  - measure
+upforgrabs:
+  name: help wanted
+  link: hhttps://github.com/WOSPM/checker/labels/help%20wanted
```
Comment (ritwik12) [inline _data/projects/wospm-checker.yml]: @umutphp This isn't a valid URL. Please remove extra `h` from the beginning.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/wospm-checker.yml (hunk 2 above)
Comment (umutphp) [reply _data/projects/wospm-checker.yml]: Sorry for that. Fixed :)

**Review by ritwik12 — APPROVED** [review]

File: _data/projects/wospm-checker.yml (hunk 1 above)
Comment (umutphp) [reply _data/projects/wospm-checker.yml]: I have changed the description a bit. Can you please check it again?

**Comment by ritwik12** [comment]
@umutphp Thanks!!

**Pull request merged**